src/courses/models.py

```diff
@@ -205,6 +205,8 @@ class Semester(models.Model):
     QUEST_AWAITING_APPROVAL = -2
     STUDENTS_WITH_NEGATIVE_XP = -3
 
+    name = models.CharField(null=True, unique=False, max_length=50)
```

Comment:
null=True, blank=False <--- that might be why you needed that extra stuff in the form?

Reply:
null <--- database can be emtpy
blank <--- form can be empty

Reply:
changed